app.css

```diff
@@ -0,0 +1,56 @@
+* {
+  direction: rtl;
+  padding:0px;
+  margin:0px;
+}
```

Comment:
This block is highly problematic - you're doing these rules on ALL elements, instead of inheriting them.
If you set this css rules block on the html & body, then it will pass to all children unless you give them a different css rule, or a third party does so.
This clashes with slick slider that doesn't expect it's slides to be RTL.